templates/Wpf/Features/XamlIslandUwpApp/.template.config/template.json

```diff
@@ -0,0 +1,87 @@
+﻿{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Universal"
+  ],
+  "name": "Xaml Island Uwp App",
+  "shortName": "Xaml Island Uwp App",
+  "identity": "wts.Wpf.Feat.XamlIslandUwpApp",
+  "groupIdentity": "wts.Wpf.Feat.XamlIslandUwpApp",
+  "description": "Infrastructure for Xaml Islands.",
+  "tags": {
```

Comment:
should wts.displayOrder also be set?

Reply:
not neccesary because it has "wts.isHidden": "true"